tests/bud.bats

```diff
@@ -2811,3 +2816,301 @@ _EOF
   ! expect_output --substring '\-\-build-arg SECRET=<VALUE>'
   ! expect_output --substring '\-\-build-arg NEWSECRET=<VALUE>'
 }
+
+@test "bud with --runtime and --runtime-flag" {
+  # This Dockerfile needs us to be able to handle a working RUN instruction.
+  skip_if_no_runtime
+  skip_if_chroot
+
+  _prefetch alpine
+
+  mytmpdir=${TESTDIR}/my-dir
+  mkdir -p ${mytmpdir}
+cat > $mytmpdir/Containerfile << _EOF
+from alpine
+run echo hello
+_EOF
+
+  local found_runtime=
+
+  if [ -n "$(command -v runc)" ]; then
+    found_runtime=y
+    run_buildah ? bud --runtime=runc --runtime-flag=debug \
+                      -q -t alpine-bud-runc --signature-policy ${TESTSDIR}/policy.json --file ${mytmpdir} .
+    if [ "$status" -eq 0 ]; then
+      expect_output --substring "nsexec started"
+    else
+      # runc fully supports cgroup v2 (unified mode) since v1.0.0-rc93.
+      # older runc doesn't work on cgroup v2.
+      expect_output --substring "this version of runc doesn't work on cgroups v2" "should fail by unsupportability for cgroupv2"
+    fi
+  fi
+
+  if [ -n "$(command -v crun)" ]; then
+    found_runtime=y
+
+    # Use seccomp to make crun output a warning message because crun writes few logs.
+    cat > ${TESTDIR}/seccomp.json << _EOF
+{
+    "defaultAction": "SCMP_ACT_ALLOW",
+    "syscalls": [
+      {
+        "name": "unknown",
+        "action": "SCMP_ACT_KILL"
+	    }
+    ]
+}
+_EOF
+
+    run_buildah bud --runtime=crun --runtime-flag=debug --security-opt seccomp=${TESTDIR}/seccomp.json \
+                    -q -t alpine-bud-crun --signature-policy ${TESTSDIR}/policy.json --file ${mytmpdir} .
+    expect_output --substring "unknown seccomp syscall"
+  fi
+
+  if [ -z "${found_runtime}" ]; then
+    skip "Did not find 'runc' nor 'crun' in \$PATH - could not run this test!"
```

Comment:
It might be better to `die` here, not `skip`: if this ever triggers, it means there's a new runtime that we need to handle.

Reply:
Done.